packages/nx/src/tasks-runner/life-cycles/formatting-utils.ts

```diff
@@ -5,5 +5,17 @@ export function formatFlags(
 ): string {
   return flag == '_'
     ? `${leftPadding}  ${(value as string[]).join(' ')}`
-    : `${leftPadding}  --${flag}=${value}`;
+    : `${leftPadding}  --${flag}=${formatValue(value)}`;
+}
+
+function formatValue(value: any) {
+  if (typeof value === 'string' || typeof value === 'number') {
```

Comment:
do we need to consider boolean values?

Reply:
Actually, string, number, and boolean are fine when just uses as `value.toString()`

Reply:
Let me change that quickly to simplify the code